pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -75,8 +86,13 @@ func (o *OssmInstaller) processManifests() error {
 		return internalError(errors.WithStack(err))
 	}
 
+	copyFsErr := copyEmbeddedFS(embeddedFiles, "templates", filepath.Join(outputDir, o.Namespace, o.Name))
+	if copyFsErr != nil {
+		return internalError(errors.WithStack(err))
+	}
+
 	for i, m := range o.manifests {
-		if err := m.processTemplate(data); err != nil {
+		if err := m.processTemplate(data, o.Name, o.Namespace); err != nil {
```

Comment:
```suggestion
		if err := m.processTemplate(data); err != nil {
```

as pointed out above, this does not belong here.

Reply:
[c356f9e](https://github.com/maistra/opendatahub-operator/pull/5/commits/c356f9e6b0a784ad078fd0026bc94b39a512140e) added here